src/containers/Profile/Contact/ContactProfile.tsx

```diff
@@ -101,7 +99,8 @@ export const ContactProfile = () => {
 
   const drawer = (
     <div className={styles.Drawer}>
-      {profileHeaders.map(({ id, name }) => {
+      {profileHeaders.map((profile: any) => {
+        const { id, name, is_active, is_default } = profile;
```

Comment:
Do we have to do this? Cant we stick to the old pattern?

```suggestion
      {profileHeaders.map(({ id, name, is_active, is_default } => {
```